configure

```diff
@@ -9125,7 +9125,7 @@ then
 			LDCXXSHARED='$(CXX) -shared'
 		else
 			LDSHARED='$(CC) -b'
-			LDCXXSHARED='$(CXX) -shared'
+			LDCXXSHARED='$(CXX) -b'
```

Comment:
This doesn't look related.

Reply:
Note those lines originate from #2519. Perhaps @datalogics-robb can comment on why `LDCXXSHARED='$(CXX) -b'` in `configure.ac` becomes `LDCXXSHARED='$(CXX) -shared'` in `configure`.

Reply:
I looked at #2519 and it looks like the changes in configure were editted manually, rather than generated from configure.ac (my two cents).

If the line is suppossed to be with -shared, then configure.ac needs to be fixed. I do not know HPUX, so cannot help.

Reply:
Sorry I've been AFK for a week, but yes this change looks good.  -shared is a gcc/g++ flag and should definitely not be in a non-gcc clause.